docs/apis/batch/libs/gelly.md

```diff
@@ -1734,3 +1734,547 @@ vertex represents a group of vertices and each edge represents a group of edges
+
+<div class="codetabs" markdown="1">
+<div data-lang="java" markdown="1">
+{% highlight java %}
+ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
+
+Graph<LongValue,NullValue,NullValue> graph = new CycleGraph(env, 5)
+    .generate();
+{% endhighlight %}
+</div>
+
+<div data-lang="scala" markdown="1">
+{% highlight scala %}
+import org.apache.flink.api.scala._
+import org.apache.flink.graph.generator.CycleGraph
+
+val env: ExecutionEnvironment = ExecutionEnvironment.getExecutionEnvironment
+
+val graph = new CycleGraph(env.getJavaEnv, 5).generate()
+{% endhighlight %}
+</div>
+</div>
+
+<svg class="graph" width="540" height="540"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:xlink="http://www.w3.org/1999/xlink">
+
+    <line x1="270" y1="40" x2="489" y2="199" />
+    <line x1="489" y1="199" x2="405" y2="456" />
+    <line x1="405" y1="456" x2="135" y2="456" />
+    <line x1="135" y1="456" x2="51" y2="199" />
+    <line x1="51" y1="199" x2="270" y2="40" />
+
+    <circle cx="270" cy="40" r="20" />
+    <text x="270" y="40">0</text>
+
+    <circle cx="489" cy="199" r="20" />
+    <text x="489" y="199">1</text>
+
+    <circle cx="405" cy="456" r="20" />
+    <text x="405" y="456">2</text>
+
+    <circle cx="135" cy="456" r="20" />
+    <text x="135" y="456">3</text>
+
+    <circle cx="51" cy="199" r="20" />
+    <text x="51" y="199">4</text>
+</svg>
+
+### Empty Graph
+
+The graph containing no edges.
+
+<div class="codetabs" markdown="1">
+<div data-lang="java" markdown="1">
+{% highlight java %}
+ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
+
+Graph<LongValue,NullValue,NullValue> graph = new EmptyGraph(env, 5)
+    .generate();
+{% endhighlight %}
+</div>
+
+<div data-lang="scala" markdown="1">
+{% highlight scala %}
+import org.apache.flink.api.scala._
+import org.apache.flink.graph.generator.EmptyGraph
+
+val env: ExecutionEnvironment = ExecutionEnvironment.getExecutionEnvironment
+
+val graph = new EmptyGraph(env.getJavaEnv, 5).generate()
+{% endhighlight %}
+</div>
+</div>
+
+<svg class="graph" width="540" height="80"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:xlink="http://www.w3.org/1999/xlink">
+
+    <circle cx="30" cy="40" r="20" />
+    <text x="30" y="40">0</text>
+
+    <circle cx="150" cy="40" r="20" />
+    <text x="150" y="40">1</text>
+
+    <circle cx="270" cy="40" r="20" />
+    <text x="270" y="40">2</text>
+
+    <circle cx="390" cy="40" r="20" />
+    <text x="390" y="40">3</text>
+
+    <circle cx="510" cy="40" r="20" />
+    <text x="510" y="40">4</text>
+</svg>
+
+### Grid Graph
+
+An undirected graph connecting vertices in a regular tiling in one or more dimensions.
+
+<div class="codetabs" markdown="1">
+<div data-lang="java" markdown="1">
+{% highlight java %}
+ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
+
+Graph<LongValue,NullValue,NullValue> graph = new GridGraph(env)
+    .addDimension(2, false)
```

Comment:
I think we should explain what `addDimension` does and what its arguments represent.